packages/contracts/deploy/40_finalize-managing-dao/00_register-managing-dao-on-dao-registry.ts

```diff
@@ -0,0 +1,72 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+
+import {getContractAddress} from '../helpers';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log(`\nFinalizing ManagingDao.`);
+
+  const {getNamedAccounts, ethers} = hre;
+  const {deployer} = await getNamedAccounts();
+
+  // Get info from .env
+  const daoSubdomain = process.env.MANAGINGDAO_SUBDOMAIN || '';
+
+  if (!daoSubdomain)
+    throw new Error('ManagingDAO subdomain has not been set in .env');
+
+  // Get `managingDAO` address.
+  const managingDAOAddress = await getContractAddress('DAO', hre);
+
+  // Get `DAO` contract.
+  const managingDaoContract = await ethers.getContractAt(
+    'DAO',
+    managingDAOAddress
+  );
+
+  // Get `DAORegistry` address.
+  const daoRegistryAddress = await getContractAddress('DAORegistry', hre);
+
+  // Get `DAORegistry` contract.
+  const daoRegistryContract = await ethers.getContractAt(
+    'DAORegistry',
+    daoRegistryAddress
+  );
+
+  // Grant `REGISTER_DAO_PERMISSION` to deployer.
+  const REGISTER_DAO_PERMISSION_ID = ethers.utils.id('REGISTER_DAO_PERMISSION');
+
+  const grantTx = await managingDaoContract.grant(
```

Comment:
Shouldn't grant be made on the registry itself? Or all these contracts are basically using the managing DAO's permission database?

Reply:
they use the Managing DAO's database